### PR TITLE
Add ESLint and ESLint's rules

### PR DESCRIPTION
**Pull request by cihat**
Hi @dashersw, I've recently become interested in [erste.js](https://github.com/cihat/erste/). I am working in the native gui area.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "semi": "error",
```
Comment (dashersw) [inline .eslintrc.js]: thank you very much! Is it possible to add the configuration under https://github.dev/coyotiv/stack? That's what I'd like to follow—including prettier and the following config:

```js
{
  "singleQuote": true,
  "arrowParens": "avoid",
  "printWidth": 120,
  "semi": false,
  "trailingComma": "es5"
}
```

Comment (cihat) [reply .eslintrc.js]: Hi @dashersw , this is how I adapted the ESLint rules for these rules. I also updated the eslintrc.js file as json.

```json
   "rules": {
     "quotes": ["error", "single"],
     "arrow-parens": ["error", "always"],
     "max-len": ["error", {
       "code": 120
     }],
     "semi": "error",
     "comma-dangle": ["error"]
   }
```

Comment (cihat) [reply .eslintrc.js]: I opened a different branch and added prettier and eslint with the config used in @coyotiv/stack.
I thought about closing this branch as well.

**Comment by cihat** [comment]
I thought about adding both Prettier and ESLint in [this](https://github.com/dashersw/erste-demo/pull/4) branch.

@dashersw

**Pull request closed without merging**